terraform-aws-github-runner/modules/runners-instances/templates/install-config-runner.sh

```diff
@@ -65,6 +65,24 @@ EOF
 . /home/$USER_NAME/runner-scripts/utils.sh
 
 sudo chown -R $USER_NAME:$USER_NAME /home/$USER_NAME/actions-runner
+
+instance_id=\$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
+region=\$(curl -s http://169.254.169.254/latest/meta-data/placement/region)
+runner_type=\$(aws ec2 describe-tags --filters "Name=resource-id,Values=\$instance_id" "Name=key,Values=RunnerType" --query 'Tags[0].Value' --output text --region \$region)
+instance_type=\$(curl -s http://169.254.169.254/latest/meta-data/instance-type)
+ami_id=\$(curl -s http://169.254.169.254/latest/meta-data/ami-id)
```

Comment:
This depends on IDMS v1, which will fail if we ever switch to v2, can we have some sort of fallback here?

Also please note that we already print this info as part of `setup Linux` job

Reply:
Good call out.  Seems like the old AMI is also compatible with the v2 interface, so I'll just use that here

Reply:
Confirming that using v2 worked. PR updated accordingly